curriculum/utils.js

```diff
@@ -83,6 +83,7 @@ const directoryToSuperblock = {
     'foundational-c-sharp-with-microsoft',
   '20-upcoming-python': 'upcoming-python',
   '21-a2-english-for-developers': 'a2-english-for-developers',
+  '22-rosetta-code': 'rosetta-code',
```

Comment:
It looks like this will need to be updated to something other than 22 since Tom is also using that the Python curriculum

https://github.com/freeCodeCamp/freeCodeCamp/pull/52628/files#diff-7dc5a3c37a34a8e58cef2f545da6da29e0b376364c3c16a85b4a46c83e7baf4f

Reply:
Thanks @jdwilkin4 - I can change it on the other PR. This one will be easier to get in.